dgbpy/torch_classes.py

```diff
@@ -746,82 +745,170 @@ def transform(self, arr):
             return np.rot90(arr,self.aug_count,self.aug_dims).copy()
 
 class RandomGaussianNoise():
-    def __init__(self, p=0.3, std=.3):
+    def __init__(self, p=0.3, std=0.1):
         self.p = p
         self.std = std
 
-    def update_pars(self, transform_dict):
-        self.p = float(transform_dict['p'])
-        self.std = float(transform_dict['std'])
+    def transformLabel(self, info):
+        return False
         
-    def __call__(self, image, label, image_only=True):
-        self.noise = np.random.normal(loc = 0, scale = self.std, size = image.shape).astype('float32')
+    def __call__(self, image=None, label=None):
         if self.p > np.random.uniform(0,1):
+            self.noise = np.random.normal(loc = 0, scale = self.std, size = image.shape).astype('float32')
             return self.transform(image), label
-        else:
-            return image, label
+        return image, label
 
     def transform(self, arr):
         arr = self.noise + arr
         return arr
 
+def hasOpenCV():
+  try:
+    import cv2
+  except ModuleNotFoundError:
+    return False
+  return True
+
+class RandomRotation():
+    def __init__(self, p=0.3, angle=15):
+        self.p = p
+        self.angle = angle
+        import cv2
```

Comment:
Would it be possible to avoid adding a dependency on OpenCV by using scipy.ndimage which is already included in the OpendTect Python distribution? There is scipy.ndimage.rotate - don't know if it does the same.